projects/ngrx.io/content/guide/router-store/selectors.md

```diff
@@ -33,22 +30,85 @@ export const {
   selectRouteParam,     // factory function to select a route param
   selectRouteData,      // select the current route data
   selectUrl,            // select the current url
-} = fromRouter.getSelectors(selectRouter);
+} = getSelectors(selectRouter);
+```
+
+`car.state.ts`
+
+```ts
+import {
+  createReducer,
+  createAction,
+  props,
+  on,
+  createFeatureSelector,
+  createSelector
+} from "@ngrx/store";
+import { EntityState, createEntityAdapter } from "@ngrx/entity";
+
+export interface Car {
+  id: string;
+  year: string;
+  make: string;
+  model: string;
+}
+
+export type CarState = EntityState<Car>;
+
+export const carAdapter = createEntityAdapter<Car>({
+  selectId: car => car.id
+});
 
-export const selectSelectedCarId = selectQueryParam('carId');
-export const selectCar = createSelector(
-   selectCarEntities,
-   selectSelectedCarId,
-   (cars, carId) => cars[carId]
+// for our example, we'll only populate cars in the store on app init
+export const appInit = createAction("[App] Init", props<{ cars: Car[] }>());
```

Comment:
I think this should be in a separate code block, just so its not confusing about inlining actions

Reply:
I've made this into a new file now - I think that this is more in line with the rest of the docs?